### PR TITLE
LAMMPS 29Oct20 foss and fosscuda

### PR DESCRIPTION
**Pull request by afraz-ahmed**
For INC1109825 & INC1112385.

Please also see https://github.com/bear-rsg/easybuild-easyblocks/pull/72

* [x] Assigned to reviewer

`LAMMPS-29Oct2020-foss-2020a-Python-3.8.2-kokkos.eb`
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL8-cascadelake
* [ ] EL8-haswell

`LAMMPS-29Oct2020-fosscuda-2020a-Python-3.8.2-kokkos.eb`
(Note for Haswell: gpu needs to be present for sanity check commands to work)
* [ ] EL8-haswell
* [ ] EL8-power9

### CODE REVIEW

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-29Oct2020-fosscuda-2020a-Python-3.8.2-kokkos.eb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    ('PCRE', '8.44'),
+    ('libxml2', '2.9.10'),
+    ('FFmpeg', '4.2.2'),
+    ('Voro++', '0.4.6'),
+    ('kim-api', '2.1.3'),
+    ('Eigen', '3.3.7'),
+    ('yaff', {'arch=x86_64': '1.6.0', 'arch=POWER': False}, local_python_versionsuffix),
+    ('PLUMED', '2.6.0', local_python_versionsuffix),
+    ('ScaFaCoS', {'arch=x86_64': '1.0.1', 'arch=POWER': False}),
+    ('VTK', {'arch=x86_64': '8.2.0', 'arch=POWER': False}, local_python_versionsuffix),
```
Comment (branfosj) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-29Oct2020-fosscuda-2020a-Python-3.8.2-kokkos.eb:61]:
```suggestion
    ('VTK', '8.2.0', local_python_versionsuffix),
```

I fixed VTK to build on POWER in #463, so we should be fine to use it as a dep here.

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-29Oct2020-fosscuda-2020a-Python-3.8.2-kokkos.eb [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    'GRANULAR', 'KIM', 'KSPACE', 'MANYBODY', 'MC', 'MESSAGE', 'MISC',
+    'MOLECULE', 'MPIIO', 'PERI', 'POEMS', 'PYTHON', 'QEQ', 'REPLICA', 'RIGID',
+    'SHOCK', 'SNAP', 'SPIN', 'SRD', 'VORONOI',
+]
+
+# not enabled (yet), needs more work/additional dependencies:
+# ADIOS - https://lammps.sandia.gov/doc/Build_extras.html#user-adios-package
+# AWPMD - https://lammps.sandia.gov/doc/Build_extras.html#user-awpmd-package
+# QUIP  - https://lammps.sandia.gov/doc/Build_extras.html#user-quip-package
+# VTK (listed below) will only be enabled if VTK is a dependency (I.e. on x86_64 only)
```
Comment (branfosj) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-29Oct2020-fosscuda-2020a-Python-3.8.2-kokkos.eb:89]:
```suggestion
```

**Review by branfosj — CHANGES_REQUESTED** [review]
A hopefully minor change. The VTK addition on POWER will need to be tested to check it still builds.

**Comment by afraz-ahmed** [comment]
VTK built successfully on Power9, although I needed to change `$EASYBUILD_TMPDIR`.

**Review by branfosj — APPROVED** [review]

**Pull request merged**